src/main/java/com/entropyteam/entropay/employees/services/EmployeeService.java

```diff
@@ -109,6 +113,9 @@ public EmployeeDto create(EmployeeDto employeeDto) {
         Employee entityToCreate = toEntity(employeeDto);
         Employee savedEntity = getRepository().save(entityToCreate);
         paymentInformationService.createPaymentsInformation(savedEntity.getPaymentsInformation(), savedEntity);
+        LocalDate birthDate = employeeDto.birthDate();
+        LocalDate modifiedBirthDate = birthDate.withYear(LocalDate.now().getYear());
+        googleService.createGoogleCalendarEvent("BirthDay " + employeeDto.firstName() + " " + employeeDto.lastName(), modifiedBirthDate, modifiedBirthDate);
```

Comment:
minor, lets extract this lines in a method that receives a list of employees, create the event name then call google service